clean_bucket/adt/1993010100.yaml

```diff
@@ -0,0 +1,111 @@
+aws_s3:
+     adt.cryosat2:
+
+          # Define the path attributes.
+          local_path: !ENV ${WORKrnr}/${EXPTrnr}/${CYCLE}/intercom/inputs/obs/ocean/cryosat2.adt.iodav2.nc
```

Comment:
I think we should remove this long path from the yaml `${WORKrnr}/${EXPTrnr}/${CYCLE}/intercom/inputs/obs/ocean/` and replace it with something more generic like `${CLEAN_BUCKET_DESTINATION_PATH}`. My thinking is that others might want to use these templates for retrieval for cases that have nothing to do with the RNR. It will be up to the workflow to then pre-fill `CLEAN_BUCKET_DESTINATION_PATH=${WORKrnr}/${EXPTrnr}/${CYCLE}/intercom/inputs/obs/ocean/`